msbuild/Xamarin.MacDev.Tasks.Core/Tasks/LinkNativeCodeTaskBase.cs

```diff
@@ -48,6 +50,9 @@ public override bool Execute ()
 					var libExtension = Path.GetExtension (lib).ToLowerInvariant ();
 					switch (libExtension) {
 					case ".a":
+						var forceLoad = string.Equals (libSpec.GetMetadata ("ForceLoad"), "true", StringComparison.OrdinalIgnoreCase);
```

Comment:
don't we already have an extended bool parsing method ? accepting yes, no, 1, 0...
otherwise maybe use `Boolean.TryParse` ?

Reply:
The string comes from MSBuild, where the convention is to use `true`/`false` (and string comparisons are done case-insensitively), which is why I did it this way. Also this isn't really a user-configurable value, we're setting it ourselves elsewhere.